### PR TITLE
Change Atmosphere Effect Sample

### PR DESCRIPTION
**Pull request by Rachael-E**
@TADraeseke please could you review this and let me know what you think? Thanks!

### CODE REVIEW

File: kotlin/change-atmosphere-effect/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## How to use the sample
+
+Select one of the three available atmosphere effects. The sky will change to display the selected atmosphere effect. 
+
+## How it works
+To change the atmosphere effect:
+
+1. Create an `ArcGISScene` and display it in a `SceneView`.
+2. Change the atmosphere effect with `sceneView.AtmosphereEffect(AtmosphereEffect)`.
```
Comment (TADraeseke) [inline kotlin/change-atmosphere-effect/README.md]:
```suggestion
2. Change the atmosphere effect with `sceneView.atmosphereEffect(AtmosphereEffect)`.
```

File: kotlin/change-atmosphere-effect/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+## Relevant API
+
+* ArcGISScene
+* AtmosphereEffect
+* SceneView
+
+## Additional information
+There are three atmosphere effect options:
+
+- **Realistic** - A realistic atmosphere effect is applied over the entire surface.
```
Comment (TADraeseke) [inline kotlin/change-atmosphere-effect/README.md]: Let's make these all caps so they look like java/kotlin enums

File: kotlin/change-atmosphere-effect/src/main/java/com/esri/arcgisruntime/sample/changeatmosphereeffect/MainActivity.kt [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+        // add base surface for elevation data
+        with(Surface()) {
+            this.elevationSources.add(
+              ArcGISTiledElevationSource(
+                      getString(R.string.elevation_image_service)
+              )
+            )
+            sceneView.scene.baseSurface = this
+        }
```
Comment (TADraeseke) [inline kotlin/change-atmosphere-effect/src/main/java/com/esri/arcgisruntime/sample/changeatmosphereeffect/MainActivity.kt]: We'll try to use the apply block when following a "builder" pattern like this:
```
    // add base surface for elevation data
    sceneView.scene.baseSurface = Surface().apply {
      elevationSources.add(ArcGISTiledElevationSource(getString(R.string.elevation_image_service)))
    }
```

File: kotlin/change-atmosphere-effect/src/main/java/com/esri/arcgisruntime/sample/changeatmosphereeffect/MainActivity.kt [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+        return super.onCreateOptionsMenu(menu)
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        // set different atmosphere effects on the scene view
+        sceneView.atmosphereEffect = when (item.itemId) {
+            R.id.action_no_atmosphere_effect -> AtmosphereEffect.NONE
+            R.id.action_realistic_atmosphere_effect -> AtmosphereEffect.REALISTIC
+            R.id.action_horizon_atmosphere_effect -> AtmosphereEffect.HORIZON_ONLY
+            else -> return super.onOptionsItemSelected(item)
```
Comment (TADraeseke) [inline kotlin/change-atmosphere-effect/src/main/java/com/esri/arcgisruntime/sample/changeatmosphereeffect/MainActivity.kt]: This is a good pattern and one the Java sample seems to do wrong. Nice catch!

**Comment by Rachael-E** [comment]
thanks @TADraeseke! I've made those changes: will go ahead and merge!

**Pull request merged**